apps/ledger-live-mobile/Gemfile.lock

```diff
@@ -278,7 +278,6 @@ GEM
 
 PLATFORMS
   ruby
-  x86_64-darwin-20
```

Comment:
🤔 not sure how it relates to this new coin addition

Reply:
It might have been staged accidentally. I am sorry about that 😓, I have reverted this in the latest commit.